app/assets/stylesheets/pages/_home.scss

```diff
@@ -1 +1,37 @@
 // Specific CSS for your home-page
+h2 {
+  text-align: center;  
+}
+
+#yacht-banner {
+  margin: auto;
+  margin-bottom: 20px;
+  position: static;  
+  background-image: url("https://images.unsplash.com/photo-1519011866958-6ac4dd2bbba4?ixid=MXwxMjA3fDB8MHxzZWFyY2h8NTl8fHNlYSUyMHBhdHRlcm58ZW58MHx8MHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=500&q=60");
```

Comment:
I would save it in the /app/assets/images folder.

Reply:
Consider it done (tomorrow hehe)